cmd/main.go

```diff
@@ -117,10 +118,10 @@ func main() {
 		cancel()
 	}()
 
-	log.Info().Str("version", conf.Admin.Version).Msg("Starting services.")
+	log.Info().Str("version", conf.Version).Msg("Starting services.")
 	sg.GoListenAndServe("api", conf.API.Port, routers.NewDefaultAPIRouter(optlyCache, conf.API, agentMetricsRegistry))
 	sg.GoListenAndServe("webhook", conf.Webhook.Port, routers.NewWebhookRouter(optlyCache, conf.Webhook))
-	sg.GoListenAndServe("admin", conf.Admin.Port, routers.NewAdminRouter(conf.Admin)) // Admin should be added last.
+	sg.GoListenAndServe("admin", conf.Admin.Port, routers.NewAdminRouter(*conf)) // Admin should be added last.
```

Comment:
Should `NewAdminRouter` accept a pointer type, instead of dereferencing it here?

Reply:
I was going back and forth on this. I wanted to be consistent with all of the the other constructors which are not using a pointer.